draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1110,6 +1110,32 @@ to determine if 0-RTT data is permissible.
 > the same connection, replayed media packets would be discarded as duplicates by
 > the receiver.
 
+## Coalescing RTP packets in single QUIC packet
+
+
+Applications usually cannot control how the QUIC stack maps application data to
+QUIC packets. While the application controls the content of QUIC DATAGRAM
+frames, stream frames may contain any amount of data. QUIC packets can include
+multiple STREAM frames from different streams and one or more DATAGRAM frames.
+Applications don't necessarily control how the QUIC stack maps STREAM and
+DATAGRAM frames to QUIC packets {{Section 13 of ?RFC9000}} and {{Section 5 of
+?RFC9308}}.
+
+If there is not enough data, QUIC stacks might wait before sending out packets
+to optimize for bandwidth utilization instead of latency, while real-time
+applications usually prefer to optimize for latency rather than bandwidth
+utilization.
+
+When there are a lot of small DATAGRAM frames (e.g., an audio stream) and a lot
+of large DATAGRAM (e.g., a video stream), it may be a good idea to make sure the
+audio frames can be included in a QUIC packet that also carries video frames
+(i.e., the video frames don't fill the whole QUIC packet). Otherwise, the QUIC
+stack may have to send additional small packets only carrying single audio
+frames, which would waste some bandwidth.
+
+Application designers are advised to take these considerations into account when
+selecting and configuring a QUIC stack for use with RoQ.
```

Comment:
I know we've talked about this before, but some application designers have control over QUIC implementations, and others do not. Is this also a reason why we want people to use "rtp-mux-quic"? If so, we should probably say that, someplace - maybe in a new subsection 7.x. 

(if your QUIC implementation accepts that ALPN string, you ought to be smart about this kind of thing)

Reply:
How is it related to the ALPN? All QUIC implementations I have worked with so far accept any ALPN string I gave them, and I don't think there is any kind of special behavior based on what the ALPN is.

Reply:
Yeah, so, that's the thing. I'm not sure whether having our own ALPN string will make any difference at all in practice. One might think that it would make a difference - for instance, a QUIC implementation might focus more on minimizing one-way delays, and not worry as much about probing for bandwidth - but if QUIC implementations don't make that adjustment, I probably need to rethink why we still need "rtp-mux-quic".

Note that this is much more about me needing to think, than us needing to change anything in the spec!

Reply:
We need "rtp-mux-quic" to tell the application on the other end that we will use the QUIC connection to send RTP packets encapsulated as described in this document.

Reply:
And now I remember that - thanks for the wake-up call!